js/views/appView.js

```diff
@@ -23,6 +23,9 @@ define([
       $("body").on("rightAnswer",function(event,data){
         _this.countDown.clear().run()
       })
+	  $("body").on("wrongAnswer",function(event,data){
```

Comment:
The $("body") "rightAnswer" already existed so I thought it would be better
to add a corresponding "wrong answer".
On Sep 8, 2015 11:50 PM, "Alex" <notifications@github.com> wrote:

> In js/views/appView.js
> <https://github.com/PhoenixJSMeetup/mathbeast/pull/14#discussion_r39011767>
> :
>
> > @@ -23,6 +23,9 @@ define([
> >        $("body").on("rightAnswer",function(event,data){
> >          _this.countDown.clear().run()
> >        })
> > +	  $("body").on("wrongAnswer",function(event,data){
>
> This could be more specific and included in the events section of the view
> so that you don't have to 'dirty' up your body and worry about things
> bubbling up.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PhoenixJSMeetup/mathbeast/pull/14/files#r39011767>.
>